packages/scss/src/components/inputs/checkbox/_checkbox.framed.scss

```diff
@@ -0,0 +1,11 @@
+%checkbox-framed {
+	@include fakeBorder(1px, _theme("commons.divider.color"));
+
+	display: flex;
+	flex-wrap: wrap;
+	padding: _component("input.framed.top-padding") _component("input.framed.side-padding") _component("input.framed.bottom-padding");
```

Comment:
Why don't we use a sigle "x x x x" padding value?

Reply:
You mean as a variable ? Because we need the individual values elsewhere.